DSA_Session5/src/employee/Main.java

```diff
@@ -0,0 +1,76 @@
+package employee;
+
+import java.util.Scanner;
+/**
+ * Driver Class
+ * 
+ * @author Aman Gautam
+ * 
+ * Dated - 11/08/2019
+ */
+public class Main{
+	
+	/**
+	 * The main method.
+	 *
+	 * @param args the arguments
+	 */
+	public static void main(String[] args) {
+
+		try {
+			LinkedList employeeList = new LinkedList();
+			Employee employeeObject;
+			Scanner input = new Scanner(System.in);
+			String employeeName;
+			int employeeAge, noOfEmployee, flag;
+			double employeeSalary;
+			while (true) {
+				System.out.println("enter no. of employee");
+				noOfEmployee = input.nextInt();
+				if (noOfEmployee > 0) {
+
+					loop:while (noOfEmployee != 0) {
```

Comment:
Need an explanation for this.

Reply:
"loop" is used as a label , in case the age of employee is <18 years then the while loop will start again , for the new input . This is done in order to validate the age input .